src/main/kotlin/com/recipebook/domain/ingredient/Ingredient.kt

```diff
@@ -0,0 +1,25 @@
+package com.recipebook.domain.ingredient
+
+import com.recipebook.domain.recipe.Recipe
+import com.recipebook.orm.AbstractJpaPersistable
+import java.io.Serializable
+import javax.persistence.Entity
+import javax.persistence.ManyToMany
+
+@Entity
+class Ingredient(var name: String, var unit: String, var quantity: Double) : AbstractJpaPersistable<Ingredient>(),
+        Serializable {
+    constructor() : this(name ="" ,unit = "", quantity = 0.00) {
+
+    }
+
+    @ManyToMany
+    val ingredients: List<Ingredient> = mutableListOf()
```

Comment:
To jest klasa Ingredient więc powinna tutaj mieć liste przepisów a nie własnych składników:) Jak to zmienisz na 
`val recipes: List<Recipe> = mutableListOf()` to też popraw metodę poniżej

Reply:
Ok